gson/pom.xml

```diff
@@ -35,7 +35,6 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-javadoc-plugin</artifactId>
         <configuration>
-          <includePackageNames>com.google.gson</includePackageNames>
```

Comment:
Was detected by Eclipse; the [`maven-javadoc-plugin`](https://maven.apache.org/plugins/maven-javadoc-plugin/javadoc-mojo.html) does not have a parameter with this name (anymore?).